catkin_ws/src/kpr_arm_control/src/arm_control/arm_control.cpp

```diff
@@ -1,14 +1,33 @@
 #include "ros/ros.h"
-#include "moveit_consts.h"
 #include <memory>
 
 #include "cucumber_msgs/CucumberContainer.h"
 #include "geometry_msgs/Pose.h"
 
+#include <ur_msgs/SetIO.h>
+
 #include <moveit/planning_interface/planning_interface.h>
+#include <moveit/planning_scene_interface/planning_scene_interface.h>
 #include <moveit/move_group_interface/move_group.h>
 
 using namespace ros;
+
+typedef std::shared_ptr<moveit::planning_interface::MoveGroup> MoveGroupPtr;
+typedef std::shared_ptr<moveit::planning_interface::PlanningSceneInterface> PlanningSceneInterfacePtr;
+
+const uint8_t VACUUM_PIN = 0x1;
+const uint8_t GRIPPER_PIN = 0x2;
+const uint8_t CUTTER_PIN = 0x4;
+
+const float GRIPPER_CLOSE = 0x1; // Not sure about value
+const float GRIPPER_OPEN = 0x0; // Not sure about value
+const float CUTTER_CLOSE = 0x1; // Not sure about value
+const float CUTTER_OPEN = 0x0; // Not sure about value
```

Comment:
Values seem okay, @alexLouw please verify this. When he approves the values, the comments should be removed.